adapters/deepintent/usersync.go

```diff
@@ -0,0 +1,13 @@
+package deepintent
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+// NewDeepintentSyncer returns deepintent syncer
+func NewDeepintentSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("deepintent", 149, temp, adapters.SyncTypeRedirect)
```

Comment:
GVL ID 149 is registered to ADman Interactive SLU. Why is this not ID 541 registered to DeepIntent, Inc.?

Reply:
corrected.